include/IECoreGL/ShaderLoader.h

```diff
@@ -43,6 +43,11 @@
 #include <map>
 #include <string>
 
+// Windows defines SearchPath
+#ifdef SearchPath
```

Comment:
This doesn't seem right - clients of SearchPath.h shouldn't have to worry about this detail. Like I said before, I think we should be able to isolate this fix to SearchPath.h. I think the key should be to figure out which Windows header creates this #define, and make sure that is included by SearchPath.h, so that we can undef it once and it'll never get defed again. Is this not possible?

Reply:
This is what I was saying before, I don't really understand why it really is needed here (also tried moving the SearchPath.h under those 2 std includes also without much luck), it affects Renderer.cpp in IECoreGL when it includes it, but TextureLoader.h which looks almost identical in terms of includes layout hasn't got this issue. My last thought is maybe SearchPath.h was included twice and the second time is skipped after it was re-defined with windows in-between somewhere, so maybe the #undef can live outside the #ifndef IE_CORE_SEARCHPATH_H which I'll try next.

Reply:
And that seemed to fix things!